client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -20,127 +20,268 @@
-  public static final String RSS_WRITER_REQUIRE_MEMORY_INTERVAL =
-      SPARK_RSS_CONFIG_PREFIX + "rss.writer.require.memory.interval";
-  public static final long RSS_WRITER_REQUIRE_MEMORY_INTERVAL_DEFAULT_VALUE = 1000; // 1s
-  public static final String RSS_COORDINATOR_QUORUM =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_COORDINATOR_QUORUM;
-  public static final String RSS_CLIENT_SEND_CHECK_TIMEOUT_MS =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_SEND_CHECK_TIMEOUT_MS;
-  public static final long RSS_CLIENT_SEND_CHECK_TIMEOUT_MS_DEFAULT_VALUE =
-      RssClientConfig.RSS_CLIENT_SEND_CHECK_TIMEOUT_MS_DEFAULT_VALUE; // 10 min
-  public static final String RSS_CLIENT_SEND_CHECK_INTERVAL_MS =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_SEND_CHECK_INTERVAL_MS;
-  public static final long RSS_CLIENT_SEND_CHECK_INTERVAL_MS_DEFAULT_VALUE =
-      RssClientConfig.RSS_CLIENT_SEND_CHECK_INTERVAL_MS_DEFAULT_VALUE;
-  public static final String RSS_TEST_FLAG = SPARK_RSS_CONFIG_PREFIX + "rss.test";
-  public static final String RSS_REMOTE_STORAGE_PATH =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_REMOTE_STORAGE_PATH;
-  public static final String RSS_INDEX_READ_LIMIT =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_INDEX_READ_LIMIT;
-  public static final int RSS_INDEX_READ_LIMIT_DEFAULT_VALUE = 500;
-  public static final String RSS_CLIENT_TYPE =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_TYPE;
-  public static final String RSS_CLIENT_TYPE_DEFAULT_VALUE = RssClientConfig.RSS_CLIENT_TYPE_DEFAULT_VALUE;
-  public static final String RSS_STORAGE_TYPE = SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_STORAGE_TYPE;
-  public static final String RSS_CLIENT_RETRY_MAX = SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_RETRY_MAX;
-  public static final int RSS_CLIENT_RETRY_MAX_DEFAULT_VALUE = RssClientConfig.RSS_CLIENT_RETRY_MAX_DEFAULT_VALUE;
-  public static final String RSS_CLIENT_RETRY_INTERVAL_MAX =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_RETRY_INTERVAL_MAX;
-  public static final long RSS_CLIENT_RETRY_INTERVAL_MAX_DEFAULT_VALUE =
-      RssClientConfig.RSS_CLIENT_RETRY_INTERVAL_MAX_DEFAULT_VALUE;
-  public static final String RSS_CLIENT_HEARTBEAT_THREAD_NUM =
-      SPARK_RSS_CONFIG_PREFIX + "rss.client.heartBeat.threadNum";
-  public static final int RSS_CLIENT_HEARTBEAT_THREAD_NUM_DEFAULT_VALUE = 4;
-  public static final String RSS_CLIENT_SEND_SIZE_LIMIT = SPARK_RSS_CONFIG_PREFIX + "rss.client.send.size.limit";
-  public static final String RSS_CLIENT_SEND_SIZE_LIMIT_DEFAULT_VALUE = "16m";
-  public static final String RSS_CLIENT_READ_BUFFER_SIZE =
-      SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_READ_BUFFER_SIZE;
+
+  public static final ConfigEntry<Integer> RSS_PARTITION_NUM_PER_RANGE = createIntegerBuilder(
+      new ConfigBuilder(SPARK_RSS_CONFIG_PREFIX + RssClientConfig.RSS_PARTITION_NUM_PER_RANGE)
+          .doc(""))
```

Comment:
If we don't have accurate description, we remove `doc`  first.

Reply:
done.